src/Pages/DistributedUptime/Details/Components/Chatbot/index.jsx

```
@@ -0,0 +1,48 @@
+// Utils
+import { useTheme } from "@emotion/react";
+
+const MESSAGES = [
+	"I've checked the network status, and we're seeing excellent performance across all regions.",
+	"The network is stable and functioning optimally. All connections are active and stable.",
+	"I've reviewed the network status, and everything looks great. No issues detected.",
+	"The network is up and running smoothly. All connections are active and stable.",
+	"I've checked the network status, and everything is looking good. No issues detected.",
+];
+
+const ChatBot = ({ sx }) => {
```

ESLint

[error] 'sx' is missing in props validation

(react/prop-types)